tests/unit/test_edp.py

```diff
@@ -57,7 +55,7 @@ def package_jats(targzfile):
     """Extract tar.gz package with JATS XML file."""
     spider = edp_spider.EDPSpider()
     response = fake_response_from_string(text="", url="file://" + targzfile)
-    return spider.handle_package_file(response).next()
+    return six.next(spider.handle_package_file(response))
```

Comment:
you don't need `six` for this if you only need compatibility with Python 2.7+, 
```suggestion
    return next(spider.handle_package_file(response))
```

Reply:
right, I see now  "The next() built-in was backported to Python 2.6+, where it calls the next method."